lib/api/readable.js

```diff
@@ -267,14 +265,35 @@ function consumeStart (consume) {
   }
 }
 
+/**
+ * @param {Buffer[]} chunks
+ * @param {number} length
+ */
+function chunksDecode (chunks, length) {
+  if (chunks.length === 0 || length === 0) {
+    return ''
+  }
+  const buffer = chunks.length === 1 ? chunks[0] : Buffer.concat(chunks, length)
+
+  const start =
+   buffer.length >= 3 &&
+   // Skip BOM.
+      buffer[0] === 0xef &&
+      buffer[1] === 0xbb &&
+      buffer[2] === 0xbf
+     ? 3
+     : 0
+  return buffer.utf8Slice(start, buffer.length - start)
```

Comment:
```suggestion
  return buffer.utf8Slice(start, buffer.length - start).toWellFormed()
```

Reply:
it was added in node v20, @tsctx you need to use the wrapper for it which falls back to toUSVString

Reply:
No idea. Just following specs.

Reply:
What is that for then? If it doesn't do anything?

Reply:
I submit to you that if you need it then there should be a corresponding test. 

@anonrig can you help?